ref/appliance-command-reference/init.md

```diff
@@ -0,0 +1,20 @@
+The `init` command configures and initialises the Anbox Cloud Appliance. This command runs through the initial bootstrap process before Anbox Cloud can be used. It provides an interactive configuration dialog which allows you to either accept the defaults or specify custom preferences for configurable aspects of the appliance. 
+
+See [Initialise the appliance](https://discourse.ubuntu.com/t/22681#initialise) for more information.
+
+## Usage
+
+    anbox-cloud-appliance init <options>
+
+## Options
+
+|Option | Description |
+|-------|-------------|
+|`--auto`|To enable automatic (non-interactive) mode. Use this option if you want to accept the default options for the appliance.|
+|`--charm-channel <channel>`|To specify the store channel from which Anbox Cloud charms are to be deployed|
+|`--force-network-bridge`|To force using a local network bridge instead of a FAN overlay network|
+|`--image-server-auth <credentials>`|To specify authentication credentials for the Anbox Cloud image server|
```

Comment:
Is there a format to specify the credentials?

Reply:
No, there isn't. Technically, the image server auth token has no format. Usually, it's just a basic auth which consists of `<user>:<password>`